data.js

```diff
@@ -0,0 +1,136 @@
+export const projects = {
+  caveat: {
+    name: "Caveat",
+    images: {
+      large: "/images/6_Hellocaveat.png",
+    },
+    logo: "/images/Caveat_White.svg",
+    href: "/info/caveat",
+    color: "blue",
+    title:
+      "Front-End, Back-End and support for the website of the creative studio Caveat",
+    url: "hellocaveat.com",
+    address: "New York, US",
+    header: "Website maintenance of creative studio Caveat",
+    stackTitle: "Stack: ",
+    stack:
+      "KeystoneJS, MongoDB, React, Typescript, GraphQL, Webpack, SCSS",
+    description:
+      ["Caveat is a creative agency based in New York. The agency is engaged in creative, copywriting, web , graphic and industrial designs. Caveat is growing and developing rapidly and the website needed to be supported and updated.","We were hired to give advice on optimizing the website infrastructure(architecture, Front-End, Back-End, deploy)"],
```

Comment:
split array items by line for `good reading` experience:
```
[
    "",
    "",
    "",
    ...
]
```

Reply:
done